eslint.config.js

```
@@ -103,6 +103,78 @@ export default [
+        showBioProfilePicturePreview: "readonly",
+        // Share globals
+        shareBioLink: "readonly",
+        copyShareBioLink: "readonly",
+        getShareBioLinkUrl: "readonly",
+        nativeShare: "readonly",
+        // Preview globals
+        getIconForUrl: "readonly",
+        getFaviconForUrl: "readonly",
+        generateBioPreviewHTML: "readonly",
+        // Reorder globals
+        setupDragAndDrop: "readonly"
```

ESLint

[error] Duplicate key 'setupDragAndDrop'.

(no-dupe-keys)